src/templates/html/application/ApplicationHTML.tsx

```diff
@@ -252,7 +255,7 @@ export function ApplicationHTML(props: {
       <head>
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <script src="https://cdn.jsdelivr.net/npm/@opensystemslab/map@0.7.5"></script>
+        <script src="https://cdn.jsdelivr.net/npm/@opensystemslab/map@0.7.9"></script>
```

Comment:
I'm certainly guilty of missing this version bump! Good catch.

Reply:
Same - surprisingly wasn't that far back!